app/controllers/laa_references_controller.rb

```diff
@@ -45,4 +63,43 @@ def no_maat_id?
   def error_messages
     @errors.map { |k, v| "#{k.humanize} #{v.join(', ')}" }.join("\n")
   end
+
+  def load_and_authorize_search
```

Comment:
**not a blocker**  repetition - could look to share this logic with `defendants_controller`